packages/webdriver/src/request/index.ts

```diff
@@ -2,33 +2,24 @@ import path from 'node:path'
 import { EventEmitter } from 'node:events'
 import { createRequire } from 'node:module'
 import { WebDriverProtocol } from '@wdio/protocols'
-import type { URL } from 'node:url'
+import { URL } from 'node:url'
 
 import logger from '@wdio/logger'
 import { transformCommandLogResult } from '@wdio/utils'
 import type { Options } from '@wdio/types'
 
-import { URLFactory } from './factory.js'
 import { isSuccessfulResponse, getErrorFromResponseBody, getTimeoutError } from '../utils.js'
 
-const require = createRequire(import.meta.url)
-const pkg = require('../../package.json')
+let pkg = { version: '' }
+if (process?.versions?.node) {
```

Comment:
Wouldn't process be undefined in the browser?

```suggestion
if ('process' in globalThis && globalThis.process.versions?.node) {
```